k8s/aat/common/ia/home-office-integration-api.yaml

```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ia-home-office-integration-api
+  namespace: ia
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: ia-home-office-integration-api
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: ia-home-office-integration-api
+    version: 0.0.2
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/ia/home-office-integration-api:prod-503867d9
+      ingressHost: ia-home-office-integration-api-aat.service.core-compute-aat.internal
+      environment:
+        RESTART_ME: "0"
```

Comment:
Not needed

Reply:
removed.  Is there another way to trigger restart of pods instead now?

Reply:
the same, just add when required